lib/msf/core/post/linux/system.rb

```diff
@@ -270,6 +300,110 @@ def get_mount_path(filepath)
   end
 
 
+  #
+  # Gets all the IP directions of the device
+  # @return [Array]
+  #
+  def ips()
```

Comment:
Style. No need to use brackets `()` in method definitions when the method takes no arguments.